app/controllers/api/v2/host_reports_controller.rb

```diff
@@ -45,6 +46,9 @@ def show
 
       def create
         @host_report = HostReport.new(host_report_params)
+        params[:host_report][:keywords]&.uniq&.each do |keyword_name|
+          @host_report.report_keywords.push(ReportKeyword.find_or_initialize_by(name: keyword_name))
```

Comment:
This is good enough for now, but I created a ticket to optimize this:

https://blog.saeloun.com/2019/11/26/rails-6-insert-all.html

https://github.com/theforeman/foreman_host_reports/projects/1#card-63314076